pkg/repositories/gormimpl/execution_repo_test.go

```diff
@@ -270,6 +270,36 @@ func TestListExecutions_Order(t *testing.T) {
 	assert.True(t, mockQuery.Triggered)
 }
 
+func TestListExecutions_WithTags(t *testing.T) {
```

Comment:
can you add a test where more than one execution is returned? For example, let's say we have:
- execution1 and tags T1 and T2
- execution2 and only tag T1
- execution3 and only tag T2

Can we have a test where the query for executions labeled with tag T1 returns execution1 and execution2, etc?

Reply:
We use mock query, so actually we won't run the query in the database.

Reply:
Can you add an integration test to https://github.com/flyteorg/flyteadmin/blob/master/tests/execution_test.go?

Reply:
added it